### DISCUSSION TITLE
google play 无法下载？

### DISCUSSION THREAD

**Discussion by fireinice** [shadowsocks/shadowsocks-android#2702] [Q&A]
大家有没有遇到过最近google play 无法下载的问题？可以登录和浏览，但是下载的时候会一直处于等待下载的状态，使用全局代理模式，dns也换了几个都是一样的问题。

**Comment by Chicago-Typer** [comment]
你的手机是不是华为或者在使用EMUI?

**Comment by fireinice** [comment]
抓包看了一下，应该是最后会被googleapis.com转回到xn--ngstr-lra8j.com，但这个地址国外ip又无法访问，而这个地址又放在了list里，没找到好的解决办法，有什么建议么？

**Reply by ThornBirds77** [reply]
我也遇到了，这个问题后来怎么解决的？

**Reply by GH-JY** [reply]
节点问题吧 这种
